apps/portal/src/content/docs/components/avatar.mdx

```diff
@@ -4,150 +4,171 @@ description: Avatar component
+</div>
+</div>`}
+/>
 
-## Anatomy
+## Usage
 
-All parts of the `Avatar` component can be imported and composed as required.
+### Image (Default)
 
 ```typescript jsx
-<Avatar.Root>
-  <Avatar.Image />
-  <Avatar.Fallback />
-</Avatar.Root>
+<Avatar name="John Doe" src="https://github.com/shadcn.png" />
+<Avatar name="John Doe" src="https://github.com/shadcn.png" size="lg" />
+<Avatar name="John Doe" src="https://github.com/shadcn.png" size="sm" />
 ```
 
-## API Reference
+### Image (Rounded)
 
-### `Root`
+```typescript jsx
+<Avatar name="John Doe" src="https://github.com/shadcn.png" rounded />
+<Avatar name="John Doe" src="https://github.com/shadcn.png" size="lg" rounded />
+<Avatar name="John Doe" src="https://github.com/shadcn.png" size="sm" rounded />
+```
 
-The `Root` component serves as the main container for all avatar elements.
+### Initials (Default)
 
 ```typescript jsx
-<Avatar.Root
-  size={6}          // [OPTIONAL] Size of the avatar, can be one of "4.5", "6", "8", or "20"
-  className="class" // [OPTIONAL] Class name to be applied to the outermost element
->
-  {/* Image and Fallback components */}
-</Avatar.Root>
+<Avatar name="John Doe" />
+<Avatar name="John Doe" size="lg" />
+<Avatar name="John Doe" size="sm" />
 ```
 
-<DocsPage.PropsTable
-  props={[
-    {
-      name: "size",
-      description:
-        "The size of the avatar. Can be one of '4.5', '6', '8', or '20'",
-      required: false,
-      value: "string",
-      defaultValue: "'6'",
-    },
-    {
-      name: "className",
-      description: "The class name to be applied to the outermost element",
-      required: false,
-      value: "string",
-    },
-  ]}
-/>
+### Initials (Rounded)
 
-### `Image`
+```typescript jsx
+<Avatar name="John Doe" rounded />
+<Avatar name="John Doe" size="lg" rounded />
+<Avatar name="John Doe" size="sm" rounded />
+```
 
-The `Image` component is used to display an image within the avatar. If the image can not be loaded, the `Fallback` component will be displayed instead.
+## API Reference
 
 ```typescript jsx
-<Avatar.Image
-  src="avatarurl.jpg" // source URL of the image
-  alt="Avatar image"  // [OPTIONAL] alt text of the image
-  className="class"   // [OPTIONAL] Class name to be applied to the image element
+<Avatar
+  name="John Doe"                     // [REQUIRED] Full name of the subject
+  src="https://github.com/shadcn.png" // [OPTIONAL] Source URL of the image
+  size="lg"                           // [OPTIONAL] Size of the avatar ('default' | 'sm' | 'lg')
+  rounded                             // [OPTIONAL] Whether the avatar is circular
 />
 ```
 
+### Props
+
 <DocsPage.PropsTable
   props={[
     {
-      name: "src",
-      description: "The source URL of the image",
+      name: "name",
+      description:
+        "Full name of the subject of the avatar. Used to generate initials for the fallback and alt text.",
       required: true,
       value: "string",
+      defaultValue: "''",
```

Comment:
Currently `name` is set as a required field, meaning it cannot have a default value.